### PR TITLE
Fixes to avoid deadlock in DTT. Example updated to use clear

### PR DESCRIPTION
**Pull request by chrthomsen**

### CODE REVIEW

File: docs/examples/testing.rst [hunk 1]
```diff
@@ -397,16 +397,20 @@ and :meth:`.setUp()`.
...
 	def setUpClass(cls):
 	    cls.cw = dtt.connectionwrapper()
-	    cls.initial = dtt.Table("book", """
+   	    cls.initial = dtt.Table("book", """
 	    | bid:int (pk) | title:text            | genre:text |
 	    | ------------ | --------------------- | ---------- |
 	    | 1            | Unknown               | Unknown    |
 	    | 2            | Nineteen Eighty-Four  | Novel      |
 	    | 3            | Calvin and Hobbes One | Comic      |
 	    | 4            | The Silver Spoon      | Cookbook   |""")
```
Comment (skejserjensen) [inline docs/examples/testing.rst:407]: Methods inside a class should be surrounded by [singular blank lines](https://peps.python.org/pep-0008/#blank-lines) according to PEP 8.

File: docs/examples/testing.rst [hunk 2]
```diff
@@ -425,8 +429,9 @@ connection from DTT on Line 4 and defines a Drawn Table with the initial state
 of the dimension in Line 5. By creating them in :meth:`.setUpClass()`, they are
 only initialized once and can be reused for each test. To ensure the tests do
 not affect each other, which would make the result depend on the execution
-order of the tests, the ``book`` table in the database is reset before each
-test by :meth:`.setUp()`. Then on Line 15 and Line 21, the tests are implemented
+order of the tests, the ``book`` table in the database is created and filled before each
+test by :meth:`.setUp()` and subsequently dropped after the test by :meth:`tearDown()`.
+Then on Line 15 and Line 21, the tests are implemented
```
Comment (skejserjensen) [inline docs/examples/testing.rst:434]: The line numbers should be updated to match the new location of the tests.

**Review by skejserjensen — CHANGES_REQUESTED** [review]

File: docs/examples/testing.rst (hunk 1 above)
Comment (chrthomsen) [reply docs/examples/testing.rst:407]: Fixed

File: docs/examples/testing.rst (hunk 2 above)
Comment (chrthomsen) [reply docs/examples/testing.rst]: Fixed

**Review by skejserjensen — APPROVED** [review]

**Pull request merged**